govtool/metadata-validation/src/main.ts

```diff
@@ -25,6 +25,7 @@ async function bootstrap() {
       exceptionFactory: () => ({ status: 200, valid: false }),
     }),
   );
+  console.log({ PORT: process.env.PORT });
```

Comment:
Is this `console.log` intentional?

Reply:
Thank you for spotting this. I removed that :saluting_face: